docs/source/schemas.rst

```diff
@@ -475,6 +475,30 @@ The ``allOf`` combiner means that any instance that is validated against
 ``software_extended-1.0.0`` will have to conform to both the base software schema
 and the properties specific to the extended schema.
 
+Default annotation
+------------------
+
+The JSON Schema spec includes a schema annotation attribute called ``default`` that
```

Comment:
Might be worth a very brief comment on why this changed, and also in that JSONSchema doesn't have well defined semantics for default either.